extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -23,20 +23,22 @@ protected function runOnPageSet( ApiPageSet $pageSet ) {
 
 			if ( is_array( $parsedInfoboxes ) && count( $parsedInfoboxes ) ) {
 				$inf = [ ];
+
 				foreach ( array_keys( $parsedInfoboxes ) as $k => $v ) {
 					$inf[ $k ] = [ ];
 				}
+
 				$pageSet->getResult()->setIndexedTagName( $inf, 'infobox' );
 				$pageSet->getResult()->addValue( [ 'query', 'pages', $id ], 'infoboxes', $inf );
-				foreach ( $parsedInfoboxes as $count => $infobox ) {
-					$sl = isset( $infobox[ 'sourcelabels' ] ) ?
-						$infobox[ 'sourcelabels' ] :
-						$this->sourceLabelsFallback( $infobox, $articleTitle );
 
+				foreach ( $parsedInfoboxes as $count => $infobox ) {
 					$pageSet->getResult()->addValue( [ 'query', 'pages', $id, 'infoboxes', $count ], 'id', $count );
-					$pageSet->getResult()->setIndexedTagName( $sl, "sourcelabels" );
+
+					$metadata = $infobox['metadata'] ?? $this->metadataFallback( $infobox, $articleTitle );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "metadata" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)